README.md

```diff
@@ -1,124 +1,83 @@
-```(bash)
-INH_URL=https://inh-admin-test.hackney.gov.uk
-AIRTABLE_API_KEY=xx
-AIRTABLE_PROMPTS_TABLE_NAME="Conversational Prompts"
-FSS_PUBLIC_API_URL=http://localhost:8085/fss
-AIRTABLE_BASE_ID=baseId123
-AIRTABLE_TABLE_NAMES="Service directory"
-AIRTABLE_BASE_URL=http://localhost:8085
-AIRTABLE_PROMPTS_API_KEY=xx
-AIRTABLE_PROMPTS_BASE_ID=baseId456
-HOTJAR_ID=12345
-```
-
-2. Build the application
-```(bash)
-yarn build
-```
-
-3. Run the tests:
-To run in cypress UI:
-
-Start the servers and run cyppress-open to launch cypress UI tool:
-
-```(bash)
-yarn start-test-servers
-yarn cypress-open
-```
-
-To run headlessly (this will launch a local application copy and mock server for you):
-
-```(bash)
-yarn int-test
-```
-
-## Deployment
-Infrastructure and code are deployed to AWS using Serverless.
-
-### Automated deployments
-Merging into `master` triggers an automated deployment into the staging environment.
-To promote this to production you will need to manually approve the deployment in CircleCI.
-
-## Pages and API endpoints
-Login is handled by Support For Hackney Residents, so long as the Hackney token cookie is set users will be signed in here too.
-
-### GET /api/snapshots/{id}
-Retrieves a vulnerabilities snapshot, given a snapshot id.
-
-### POST /api/snapshots/find
-Used by other applications (such as Support For Hackney Residents) to find snapshots related to a particular person given their name, an optionally an array of identifiers.
-
-### POST /api/snapshots
-Creates a new, empty, vulnerabilities snapshot for a specified person.
-
-### /loggedout
-The page displayed when a user is logged out.
-
-### /snapshots/{id}
-Displays a vulnerability snapshot, if there is no data saved this will display the edit view - else it displays a readonly view of the snapshot.
+# Better Conversations
+
+This tool was designed and built to enable a variety of staff across the council (Customer Services, Social Care
+Practitioners) and external workers (Voluntary Support Network, GPs, etc) to have meaningful conversations with
+residents about topics that may be out of their day-to-day remit.
+
+Better Conversations provides the ability to search for conversational prompts on certain topics, such as "Coronavirus",
+"Food", "Health", and "Benefits". The prompts are curated in an Airtable to provide the resident with the best experience
+no matter the background of the person they're talking to.
+
+Users can refer residents directly to services where the service accepts referrals via email, as well as providing
+receipts to the resident.
+
+## Intended Users & Authorisation
+
+* Hackney Customer Services can use the tool to have meaningful conversations on inbound or outbound calls.
+* Voluntary Sector Workers can discover and refer residents to other services, or council services. 
+
+There are types of authorisation groups that are permitted access to the tool, these are configured using environment
+variables:
+
+* `ALLOWED_GROUPS` - 
```

Comment:
@elena-vi / @kosiakkatrina I'm a little out of the loop on the differences between these env vars - can you give me a bit of detail about the differences please?